packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -19,9 +21,9 @@ const Redirect = ({
         case 'signup': {
             const device_data = {
                 affiliate_token   : url_params.get('affiliate_token') || '',
-                date_first_contact: url_params.get('date_first_contact') || '',
+                date_first_contact: url_params.get('date_first_contact') || getServerTime().get().format('YYYY-MM-DD'),
```

Comment:
This wouldn't work. `getServerTime` should just be `server_time`, which is an observable from the common store, you don't need the `get()` as it's not an instance of the `ServerTime` object but rather the returned moment object, so you can just rename `getServerTime` to `server_time`, and use it as `server_time.format('YYYY-MM-DD')`.